en/docs/guides/login/query-saml-assertions.md

```diff
@@ -2,3 +2,63 @@
 
 This page guides you through querying dynamic or existing [SAML2 assertions](insertlink) using standard request messages via WSO2 Identity Server. 
```

Comment:
Shall we include this information as well? "Due to some reason if you can't pass a SAML Assertion to the backend, then you can pass an identifier {link to all the identifiers that can be used to query the profile} with the request to query and obtain the assertion from the backend". Please use your wordings and include this information